src/models/BibDetails.ts

```diff
@@ -3,18 +3,39 @@ import type {
   LinkedBibDetail,
   BibDetail,
   FieldMapping,
+  AnnotatedMarcField,
+  Url,
+  SubjectHeadingDetail,
+  AnnotatedMarc,
+  AnyBibDetail,
 } from "../types/bibDetailsTypes"
 
-interface BibDetails extends Bib {
-  groupedNotes: object
-}
 export default class BibDetailsModel {
-  bib: BibDetails
-  constructor(bib: Bib) {
+  bib: Bib
+  annotatedMarcFields: AnnotatedMarcField[]
+  constructor(bib: Bib, annotatedMarc?: AnnotatedMarc) {
     this.bib = this.matchParallelToPrimaryValues(bib)
+    this.annotatedMarcFields = annotatedMarc?.fields
   }
 
-  get holdingsDetails() {
+  get annotatedMarcDetails(): AnyBibDetail[] {
+    return this.annotatedMarcFields?.map(
+      ({ label, values }: AnnotatedMarcField) => {
+        if (label === "Connect to:") {
+          const urlValues = values.map(({ label, content }) => ({
+            url: content,
+            urlLabel: label,
+          }))
+          return this.buildExternalLinkedDetail("Connect to:", urlValues)
+        } else {
+          const fieldValues = values.map((val) => val.content)
+          return this.buildDetail(label, fieldValues)
+        }
+      }
+    )
+  }
+
+  get holdingsDetails(): BibDetail[] {
```

Comment:
I think my note about annotatedMarcDetails can apply to the "details" getters as well.